dashboard/app/views/lti/v1/_early_access_banner.html.haml

```diff
@@ -0,0 +1,48 @@
+%div#lti-early-access-banner
```

Comment:
I'm on the fence about all this UI and javascript being in a HAML file. I'm worried it is missing out on all the processing and linting done by our build process in `/apps`. What do you think?